index.html

```diff
@@ -89,126 +89,47 @@ <h2 class="headline-connect">LET'S CONNECT</h2>
       </div>
     </section>
     <!-- Work Section Container-->
-    <section id="Portfolio" class="work-section">
-      <!-- Work Section 1-->
-      <div class="work-info1">
-        <div class="image-container">
-          <img
-            src="./images/work-image1.png"
-            alt="tonic work content1"
-            class="work-img"
-          />
-        </div>
-
-        <div class="content1">
-          <h3 class="work-title">Tonic</h3>
-          <h4 class="profession">
-            CANOPY <span class="dot tag"> Back End dev </span>
-            <span class="dot tag">2015</span>
-          </h4>
-          <p class="title-description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="stack-group">
-            <li class="stack">html</li>
-            <li class="stack">css</li>
-            <li class="stack">javascript</li>
-          </ul>
-          <button type="button" class="btn">See Project</button>
-        </div>
-      </div>
-
-      <!-- Work Section 2 -->
-
-      <div class="work-info2">
-        <div class="image-container">
-          <img
-            src="./images/work-image2.png"
-            alt="multi work content1"
-            class="work-img"
-          />
-        </div>
-        <div class="content1">
-          <h3 class="work-title">
-            Multi-Post <br />
-            Stories
-          </h3>
-          <h4>
-            CANOPY <span class="dot tag"> Back End dev </span>
-            <span class="dot tag">2015</span>
-          </h4>
-          <p class="title-description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="stack-group">
-            <li class="stack">html</li>
-            <li class="stack">css</li>
-            <li class="stack">javascript</li>
-          </ul>
-          <button type="button" class="btn">See Project</button>
-        </div>
-      </div>
-
-      <!-- Work Section -->
-
-      <div class="work-info3">
-        <div class="image-container">
-          <img
-            src="./images/work-image3.png"
-            alt="tonic work content2"
-            class="work-img"
-          />
-        </div>
+    <div id="cards" class="work-section"></div>
+    <section id="card-modal" class="modal-container">
+      <div class="modal">
+        <h3 class="work-title" id="modal-title">Tonic</h3>
+        <span onclick="onModalClose()" class="btn-x" draggable="true"
+          >&times;</span
+        >
```

Comment:
- [ **OPTIONAL** ] **RECOMENDED:** For JavaScript best practices I suggest you define a variable to get the **&times;(close popup)** button the use an `AddEventListener()` method to manipulate the click action.